framework/src/meshgenerators/MeshDiagnosticsGenerator.C

```diff
@@ -191,6 +191,76 @@ MeshDiagnosticsGenerator::checkSidesetsOrientation(const std::unique_ptr<MeshBas
                 " is consistently oriented with regards to the blocks it neighbors";
 
     diagnosticsLog(message, _check_sidesets_orientation, flipped_pairs.size());
+
+    // Now check that there is no sideset radically flipping from one side to another
+    // We'll consider pi / 2 to be the most steep angle we'll pass
+    unsigned int num_normals_flipping = 0;
+    Real steepest_side_angles = 1;
+    for (const auto index : index_range(side_tuples))
+    {
+      if (std::get<2>(side_tuples[index]) != bid)
```

Comment:
You use all the items in the tuple, so let's use a structured binding instead of `std::get` to make things a lot more readable

Reply:
done